api/src/changeRequests/processChangeRequest.spec.ts

```diff
@@ -30,7 +30,7 @@ describe("processChangeRequest", () => {
             doc: {},
         };
 
-        await processChangeRequest(changeRequest, new AccessMap(), db).catch((err) => {
+        await processChangeRequest("", changeRequest, new AccessMap(), db).catch((err) => {
             expect(err.message).toBeTruthy();
```

Comment:
Maybe you want to check a specific message here? Otherwise the test could be green even though there is an error for a different reason. Although I understand you're not really testing the validation here. Maybe add a spy check?

Reply:
This seems to be quite a big flaw with the current DTO validation. Will fix with the next push

Reply:
Made the test more specific